myceliald/src/constructors/typecast_transformer.rs

```diff
@@ -0,0 +1,52 @@
+use pipe::{config::Map, types::DynSection};
+use section::{command_channel::SectionChannel, message::DataType, SectionError};
+
+pub fn transformer<S: SectionChannel>(
+    config: &Map,
+) -> Result<Box<dyn DynSection<S>>, SectionError> {
+    let column = config
+        .get("column")
+        .ok_or("tagging requires a 'column' name")?
```

Comment:
nit: Some of these lines still say "tagging" but other spots have been updated to say "typecasting"

Reply:
Thank you!